Subject: lszcrypt: fix date and wrong indentation
From: Harald Freudenberger <freude@linux.vnet.ibm.com>

Summary:     s390-tools: Exploitation Support for CEX6S
Description: Exploitation Support for CEX6S
Upstream-ID: 4ad5e29f2f02e02c772ca4707b9f10253b1e5692
Problem-ID:  SEC1519

Upstream-Description:

             lszcrypt: fix date and wrong indentation

             The man page date was AUG 2008. Changed to OCT 2017.
             A previous commit had a wrong indentation on following
             options text for lszcrypt. Fixed.

             Signed-off-by: Harald Freudenberger <freude@linux.vnet.ibm.com>
             Signed-off-by: Stefan Haberland <sth@linux.vnet.ibm.com>


Signed-off-by: Harald Freudenberger <freude@linux.vnet.ibm.com>
---
 zconf/zcrypt/chzcrypt.8 |    2 +-
 zconf/zcrypt/lszcrypt.8 |    3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

--- a/zconf/zcrypt/chzcrypt.8
+++ b/zconf/zcrypt/chzcrypt.8
@@ -2,7 +2,7 @@
 .\" s390-tools is free software; you can redistribute it and/or modify
 .\" it under the terms of the MIT license. See LICENSE for details.
 .\"
-.TH CHZCRYPT 8 "AUG 2008" "s390-tools"
+.TH CHZCRYPT 8 "OCT 2017" "s390-tools"
 .SH NAME
 chzcrypt \- modify zcrypt configuration
 .SH SYNOPSIS
--- a/zconf/zcrypt/lszcrypt.8
+++ b/zconf/zcrypt/lszcrypt.8
@@ -10,7 +10,7 @@
 .\"   nroff -man lszcrypt.8
 .\" to process this source
 .\"
-.TH LSZCRYPT 8 "AUG 2008" "s390-tools"
+.TH LSZCRYPT 8 "OCT 2017" "s390-tools"
 .SH NAME
 lszcrypt \- display zcrypt device and configuration information
 .SH SYNOPSIS
@@ -91,6 +91,7 @@ The CCA Secure Key capability may be lim
 layer. The remarks 'full function set' or 'restricted function set' may
 reflect this. For details about these limitations please check the
 hypervisor documentation.
+.RE
 .TP 8
 .B -d, --domains
 Shows the usage and control domains of the cryptographic devices.
